### PR TITLE
Add resolve CLI subcommand

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Review by MartinBasti — APPROVED** [review]

File: operator_manifest/cli.py [hunk 1]
```diff
@@ -25,22 +25,28 @@ def main(args=None):
...
+        resolve_image_references(
+            namespace.images_file, authfile=namespace.authfile, output=namespace.output
+        )
     elif namespace.command == 'replace':
         replace_image_references(
             namespace.manifest_dir, namespace.replacements_file, dry_run=namespace.dry_run
         )
     elif namespace.command == 'pin':
-        # pin_image_references requires that the output_replace parameter is a seekable file and
+        # pin_image_references requires that the output_i* parameters are each a seekable file and
```
Comment (chmeliik) [inline operator_manifest/cli.py:37]:
```suggestion
        # pin_image_references requires that the output_* parameters are each a seekable file and
```

File: operator_manifest/cli.py [hunk 2]
```diff
@@ -71,6 +77,38 @@ def _make_parser():
...
+        ),
+    )
+    resolve_parser.add_argument(
+        'images_file',
+        metavar='IMAGES_FILE',
+        type=argparse.FileType('r'),
+        help=(
+            'The path to the file containing the image references to be resolved. The format of'
+            ' this file is a JSON Array of Strings where each item is an image reference. Use -'
+            ' to spcify stdin.'
```
Comment (mkosiarc) [inline operator_manifest/cli.py:93]: typo

Comment (chmeliik) [inline operator_manifest/cli.py:93]:
```suggestion
            ' to specify stdin.'
```

**Review by mkosiarc — APPROVED** [review]

File: operator_manifest/cli.py [hunk 3]
```diff
@@ -174,6 +212,30 @@ def extract_image_references(manifest_dir, output):
...
+    replacements = {}
+    for reference in references:
+        # Skip pinning of image references that already use digest
+        if '@' in reference:
+            continue
+        replacements[reference] = resolve_image_reference(reference, authfile=authfile)
+
+    json.dump(replacements, output)
+
+    return output
```
Comment (chmeliik) [inline operator_manifest/cli.py:236]: The `rtype` suggests replacements will be returned

**Review by chmeliik — COMMENTED** [review]
Thanks for adding this :tada: 

Found some typos :-)

**Comment by lcarva** [comment]
@chmeliik, @mkosiarc addressed the typos

**Review by chmeliik — APPROVED** [review]

**Pull request merged**